package.json

```diff
@@ -1,43 +1,64 @@
 {
-  "name": "bel",
-  "version": "5.1.7",
-  "description": "A simple extension to native elements",
-  "main": "index.js",
-  "browser": "browser.js",
+  "name": "nanohtml",
+  "description": "HTML template strings renderer for Node & Browsers.",
```

Comment:
`HTML template string renderer for the browser & Node`

i think we should display preference towards the browser here